pkg/controllers/nsxserviceaccount/nsxserviceaccount_controller_test.go

```diff
@@ -513,6 +513,12 @@ func TestNSXServiceAccountReconciler_Reconcile(t *testing.T) {
 				defer patches.Reset()
 			}
 
+			patches2 := gomonkey.ApplyMethod(reflect.TypeOf(r.Service), "ListNSXServiceAccountRealization",
```

Comment:
Mock `GarbageCollector` here. Otherwise you need to close `cancel chan bool` for every test case.

Reply:
ok Thanks~~